esmtools/prediction.py

```diff
@@ -735,12 +810,16 @@ def compute_persistence(reference, nlags, metric='pearson_r'):
     """
     _check_xarray(reference)
     metric = _get_metric_function(metric)
+    if metric not in [_pearson_r, _rmse]:
+        raise ValueError("""Please select between the following metrics:
```

Comment:
In the future we probably just try to get a MSE function like in xskillscore. And then I can get rid of my complicated other version.

Reply:
Agreed. I just have persistence and reconstruction compute working for pearson_r and rmse. We can worry about other metrics later and bring in the other packages you've listed.